user/src/com/google/gwt/core/server/StackTraceDeobfuscator.java

```diff
@@ -199,6 +200,17 @@ public final StackTraceElement[] resymbolize(StackTraceElement[] st, String stro
     return newSt;
   }
 
+  /**
+   * Helper to clean up client-provided symbols.
+   */
+  private static String normalizeSymbol(String symbol) {
+    // Chrome prefixes "new " for constructor calls, eliminate the prefix
+    if (symbol.startsWith("new ")) {
```

Comment:
This could also be done in the client StackTraceCreator.ModernCollector, but it seemed lower impact to include it here instead.